travis-ymls/pandas.travis.yml

```diff
@@ -0,0 +1,96 @@
+# -------------------------------------------------------------------------------------------------------------------------------------------------------
+#
+# Package             : pandas
+# Source Repo         : https://github.com/pandas-dev/pandas
+# Travis Job Link     : https://travis-ci.com/github/asellappen/pandas/
```

Comment:
if you referring  to sudo issue  then It seems sometimes appearing only, not specific to yml, seems vm environment, pls refer 


https://travis-ci.com/github/asellappen/pandas/jobs/492233393
https://travis-ci.com/github/asellappen/pandas/jobs/492233392

Reply:
In both these links, there is error saying "The job exceeded the maximum time limit for jobs, and has been terminated"

Reply:
Yes ,I have already mentioned in the last meeting , This is already existing issue on travis org in panda ,It takes more than 1 hr  and 15 mins in travis org  for amd64 and not retuning the status ,This could be the reason they have made the job to allow failure .,The same issue can be find at https://travis-ci.org/github/pandas-dev/pandas/builds/752455801 (old runs).

I tried with  -n 4  ,-n2 etc... (multiple threads) and no use, takes more than 50 mins and failing.

@Siddhesh-Ghadi ,Can you suggest what could be done here ?.

Reply:
I tried your fork on power-vm on travis. It fails with same timeout error. I think we will have to skip this one as its take more than 1hr on vm there is no way we can have it enabled on travis.

Reply:
We will have to find a way to divide the job into stages, but looking at the travis.yml it will be a difficult task.

Reply:
@Siddhesh-Ghadi ,As discussed with you ,I have tried with pytest for ppc64le instead of their own test script  ci/run_tests.sh(remainding steps are same) ,
The  job for ppc64le competed successfully with 91820 passed, 14080 skipped, 905 xfailed, 4 xpassed, 4 warnings.If it is ok then we can merge, otherwise close this one out.

Reply:
Even though the job is passing we still have `905 xfailed,` tests. I think we should mark this fail and mention in the comments cannot enable due to time limit or something like this.